### PR TITLE
Updatepartentity

### PR DESCRIPTION
**Pull request by nibor51**

### CODE REVIEW

File: src/Entity/Part.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 use App\Repository\PartRepository;
 use Doctrine\ORM\Mapping as ORM;
+use phpDocumentor\Reflection\Types\Integer;
```
Comment (rocambille) [inline src/Entity/Part.php]: is this really used?

File: templates/vehicle/parts.html.twig [hunk 2]
```diff
@@ -34,7 +34,7 @@
                             {{ include('vehicle/_delete_parts_form.html.twig') }}
                         </div>
                 </div>
-                <progress class="progress-part" value = "{{ vehicle.usedHour }}" max = "{{ part.lifespan }}"> </progress>
+                <progress class="progress-part" value = "{{ vehicle.usedHour - part.PartUseTime }}" max = "{{ part.lifespan }}"> </progress>
```
Comment (rocambille) [inline templates/vehicle/parts.html.twig]: partUseTime, not PartUseTIme

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/Entity/Part.php (hunk 1 above)
Comment (nibor51) [reply src/Entity/Part.php]: no i fix it

**Review by nibor51 — COMMENTED** [review]
I fixed what you told me

**Pull request merged**